Makefile

```
@@ -121,6 +121,9 @@ test_fast:: build get_schemas
 
 test_all:: test_pkg test_integration
 
+test_lifecycle: get_schemas
```

Does this need the schemas?  I thought that was mostly for codegen tests.
```suggestion
test_lifecycle:
```